packages/components/chip/src/ChipClearButton.tsx

```diff
@@ -41,7 +41,7 @@ export const ChipClearButton = forwardRef<HTMLSpanElement, ChipClearButtonProps>
     return (
       <span
         className={chipClearButtonWrapperStyles({
-          isBordered: design === 'dashed',
+          isBordered: ['outline', 'dashed'].includes(`${design}`),
```

Comment:
`includes(design)` instead ?

Reply:
design can be undefined and the typechecker gets wrong